packages/aws-cdk-lib/aws-codedeploy/lib/server/deployment-group.ts

```diff
@@ -369,26 +386,30 @@ export class ServerDeploymentGroup extends DeploymentGroupBase implements IServe
     }
   }
 
-  private loadBalancerInfo(loadBalancer?: LoadBalancer):
+  private loadBalancersInfo(loadBalancers?: LoadBalancer[]):
   CfnDeploymentGroup.LoadBalancerInfoProperty | undefined {
-    if (!loadBalancer) {
+
+    if (!loadBalancers) {
       return undefined;
     }
 
-    switch (loadBalancer.generation) {
-      case LoadBalancerGeneration.FIRST:
-        return {
-          elbInfoList: [
-            { name: loadBalancer.name },
-          ],
-        };
-      case LoadBalancerGeneration.SECOND:
-        return {
-          targetGroupInfoList: [
-            { name: loadBalancer.name },
-          ],
-        };
-    }
+    return loadBalancers.reduce((accumulator : {
```

Comment:
```suggestion
    return loadBalancers?.reduce((accumulator : {
```
After `if` statement removal.